wt-status.c

```diff
@@ -433,6 +516,9 @@ static void wt_status_collect_changed_cb(struct diff_queue_struct *q,
 		d->dirty_submodule = p->two->dirty_submodule;
 		if (S_ISGITLINK(p->two->mode))
 			d->new_submodule_commits = !!hashcmp(p->one->sha1, p->two->sha1);
+
+		if (s->vvp.enabled)
```

Comment:
Is there an additional benefit of the `enabled` field, or would it be equivalent to write

```c
if (s->verbose > 1)
...
```

instead? If it is equivalent, I think I would like to avoid the extra field.